app/utils/AddAssetViewHelper.scala

```diff
@@ -111,4 +112,13 @@ class AddAssetViewHelper(userAnswers: UserAnswers, mode: Mode, draftId: String)(
     )
   }
 
+  private def parseBusiness(bvm: BusinessAssetViewModel, index: Int) : AddRow = {
+    AddRow(
+      bvm.name,
+      bvm.`type`.toString,
+      business.routes.BusinessNameController.onPageLoad(mode, index, draftId).url,
```

Comment:
Should the AddRow change link not take them back to check your answers, not the business name page?

Reply:
@adamconder I'll do this as a separate PR as it applies to all the types. I think the reason I'd implemented it like this is because the asset might be 'In Progress', so there may not be any answers to render. However, I suspect most of the check your answers controllers have a required action on the name or description or whatever so it'll redirect back to that page anyway